<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="mplvl">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QListWidget" name="mplfigs">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>367</y>
      <width>200</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_simulation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBack">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPause">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Computation control:</string>
    </property>
    <widget class="QFrame" name="interact">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of excitation:</string>
      </property>
     </widget>
     <zorder>spinBox_3</zorder>
     <zorder>label_7</zorder>
     <zorder>interact_3</zorder>
    </widget>
    <widget class="QFrame" name="interact_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-60</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial position soliton:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of oscilations:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of symmetric solitons:</string>
      </property>
     </widget>
     <zorder>horizontalSlider</zorder>
     <zorder>spinBox</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>spinBox_2</zorder>
     <zorder>label_6</zorder>
     <zorder>interact_3</zorder>
     <zorder>interact_3</zorder>
     <zorder>interact_3</zorder>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>21</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="interact_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-60</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of solitons in motion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of oscilations:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of solitons in chair:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial position of solitons in motion</string>
      </property>
     </widget>
     <zorder>horizontalSlider_2</zorder>
     <zorder>spinBox_4</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>spinBox_5</zorder>
     <zorder>label_11</zorder>
     <zorder>spinBox_6</zorder>
     <zorder>label_12</zorder>
     <zorder>mplfigs</zorder>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Select module:</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Lineal continuation</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dark solitons study</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Newton's cradle</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>mplfigs</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>mplwindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_simulation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>interact_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>interact</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>interact_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>interact_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>interact</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>interact_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>interact</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>interact_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>interact_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
